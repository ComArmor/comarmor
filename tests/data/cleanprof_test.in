# A simple test comment which will persist
#include <tunables/global>

/simple/cleanprof/test/profile {
	# Just for the heck of it, this comment wont see the day of light
	#include <abstractions/base>

	#Below rule comes from abstractions/base
	allow topic /tf_static/locale/**  s,
	allow topic /node/*/** s,

	allow topic /node/foo/bar s,
	allow topic /node/foo/** p,
}

/other/cleanprof/test/profile {
	# This one shouldn't be affected by the processing
	# However this comment will be wiped, need to change that
	allow topic /node/*/** sp,
	allow topic /node/foo/bar s,
}
