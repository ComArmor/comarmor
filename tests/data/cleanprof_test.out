#include <tunables/global>

# A simple test comment which will persist


/simple/cleanprof/test/profile {
  #include <abstractions/base>

  allow topic /node/*/** s,
  allow topic /node/foo/** p,

}
/other/cleanprof/test/profile {
  allow /node/*/** sp,
  allow /node/foo/bar s,

}
